src/client/pages/OfferNew/Tabs/Tab.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { UnderlineComponent } from './Underline'
+
+const TabContainer = styled.div<{
+  selected?: boolean
+  focused?: boolean
+}>`
+  color: ${({ selected }) => (selected ? colorsV3.gray900 : colorsV3.gray500)};
+  cursor: pointer;
+  display: flex;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  padding-bottom: 0.25rem;
+  position: relative;
+  text-transform: capitalize;
+  margin-right: 1rem;
+
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    font-size: 1.5rem;
+    line-height: 2rem;
+    margin-right: 2.5rem;
+  }
+
+  &:hover {
+    color: ${colorsV3.gray900};
+  }
+`
+
+type Props = {
+  onClick: () => void
+  selected?: boolean
+  name: string
+}
+
+export const Tab: React.FC<Props> = ({ onClick, selected, name }) => {
+  const ref = React.useRef<HTMLDivElement>()
+
+  const scroll = () => {
+    onClick()
+    if (ref.current) {
+      ref.current.scrollIntoView({
+        behavior: 'smooth',
+        block: 'nearest',
+        inline: 'center',
+      })
+    }
```

Comment:
What is this supposed to do? Do we want to scroll the page when you tab so another section?